fix/stuck-instruments.lua

```diff
@@ -0,0 +1,65 @@
+-- Fixes instruments that never got played during a performance
+
+local help = [====[
+
+fix/stuck-instruments
+=====================
+
+Fixes instruments that were picked up for a performance, but were instead
+simulated and are now stuck permanently in a job that no longer exists.
+
+This works around the issue encountered with :bug:`9485`, and should be run
+if you notice any instruments lying on the ground that seem to be stuck in a
+job.
+
+Run ``fix/stuck-instruments -n`` or ``fix/stuck-instruments --dry-run`` to
+list how many instruments would be fixed without performing the action.
+
+]====]
+
+
+function fixInstruments(args)
```

Comment:
this looks like this script could be run periodically to keep the fort healthy. Would it be a good idea to add an entry to the `REPEATS` table in `gui/control-panel`? You could add that to this PR if you agree.